index.php

```diff
@@ -634,7 +634,12 @@ public function extractDownload() {
 		$downloadedVersion = $this->getVersionByVersionFile(dirname($downloadedFilePath) . '/nextcloud/version.php');
 		$currentVersion = $this->getVersionByVersionFile($this->baseDir . '/../version.php');
 		if(version_compare($downloadedVersion, $currentVersion, '<')) {
-			throw new \Exception('Downloaded version is lower than installed version');
+			// If the version is not lower, only the same major and minor are allowed
```

Comment:
What, this is also the code from Updater.php so what people run when they manually update